src/main/java/javaserver/Responders/FormResponder.java

```diff
@@ -0,0 +1,45 @@
+package javaserver.Responders;
+
+import javaserver.Form;
+import javaserver.HTTPStatusCode;
+import javaserver.Request;
+import javaserver.Response;
+
+public class FormResponder implements Responder {
+
+	private String[] supportedMethods;
+	private Form form;
+
+	public FormResponder(String[] supportedMethods, Form form) {
+		this.supportedMethods = supportedMethods;
+		this.form = form;
+	}
+
+	@Override
+	public Response getResponse(Request request) {
+		return new Response.ResponseBuilder(getStatusLine(request))
+      .body(getData(request))
```

Comment:
Oops, when I suggested that the indentation is weird on these returns, I meant to say that the chained method calls should be indented in more. Something like this:

```java
return new Response.ResponseBuilder(getStatusLine(request))
    .body(getData(request))
    .build();
```